SPECS/LICENSES-AND-NOTICES/data/licenses.json

```diff
@@ -0,0 +1,780 @@
+{
+    "header": "The CBL-Mariner SPEC files originated from a variety of sources with varying licenses.  The changelog for each SPEC file declares provenance for CBL-Mariner.  Some of the SPEC files that originated from other distributions might already have been derived from alternate sources. Their licenses can be found in LICENSE-EXCEPTION.xxx files in this folder.",
+    "table_headers": [
+        "Origin",
+        "License",
+        "Specs"
+    ],
+    "licenses": [
+        {
+            "origin": "Fedora",
+            "description": "[Fedora MIT License Declaration](https://fedoraproject.org/wiki/Licensing:Main?rd=Licensing#License_of_Fedora_SPEC_Files)",
```

Comment:
I think it would be a bit more clear, if the fields' names here matched the table headers' names above. Or in other words: could we use "license" instead of "description"?

Reply:
Done, thanks!